apps/openmw/mwgui/inventorywindow.cpp

```diff
@@ -528,14 +525,26 @@ namespace MWGui
         if (mDragAndDrop->mIsOnDragAndDrop)
         {
             MWWorld::Ptr ptr = mDragAndDrop->mItem.mBase;
+            int itemType = ptr.getContainerStore()->getType(ptr);
```

Comment:
I would have added check against ContainerStore being null to be sure, or just get the type from the MWWorld::Ptr to begin with.

Of course, if we don't check for types anymore and this line gets removed, then this comment is moot.